src/defrag.c

```diff
@@ -1139,10 +1138,12 @@ void activeDefragCycle(void) {
                 size_t frag_bytes;
                 float frag_pct = getAllocatorFragmentation(&frag_bytes);
                 serverLog(LL_VERBOSE,
-                    "Active defrag done in %dms, reallocated=%d, frag=%.0f%%, frag_bytes=%zu",
-                    (int)((now - start_scan)/1000), (int)(server.stat_active_defrag_hits - start_stat), frag_pct, frag_bytes);
+                    "Active defrag done in %dms, hits=%d, misses=%d, scanned=%d, frag=%.0f%%, frag_bytes=%zu",
+                    (int)((now - start_time)/1000), (int)(server.stat_active_defrag_hits - start_hit), 
```

Comment:
This is the changed log info.

Is it obvious that "hits=N" means the same as the old "reallocated=N"? Maybe we can keep the text "reallocated" for the hits for minimum confusion?

Is there any documentation or help message for active defrag? It would be good to explain this output somewhere.

Reply:
Okay, I think maintaining compatibility is a very important thing, but I didn't realize this problem before.
As for the output document, the logic here is already relatively low-level, I think proper annotation is a good way, because people who need this part of the log will definitely return to this code.

Reply:
But in fact I think the code itself can explain the role of the log, comments are not so necessary.

Reply:
It's fine by me, but I don't want to make the decision. :-) I highlighted this so that other reviewers can see it and decide.

Reply:
Ok, thank you very much for your reply:-)